core-bundle/src/Resources/contao/library/Contao/StringUtil.php

```diff
@@ -850,7 +850,7 @@ public static function standardize($strString, $blnPreserveUppercase=false)
 	 * @param mixed   $varValue      The serialized string
 	 * @param boolean $blnForceArray True to always return an array
 	 *
-	 * @return array|string|null The array, an empty string or null
+	 * @return mixed the unprocessed input or unserialized/empty array
```

Comment:
Seems unrelated, too.

Reply:
That's needed if we want to support serialized strings in the `ImageFactory` and `PictureFactory`. Otherwise we have to check `is_string()` there before calling `StringUtil::deserialize()` (as there could be literally anything passed in and the function).

Reply:
But `@return array|string|null` already includes serialized strings?

Reply:
Yes, but not objects. :slightly_smiling_face:

Reply:
Then it should probably be `@return array|string|object|null`? Because `mixed` also includes `int`, `bool` and `float` and that does not seem correct.

Reply:
Adjusted in f4e443c2ea38c162441edff183b893d03aaa26ed.

Reply:
It could be `int` in the `ImageFactory` and `PictureFactory` use case :see_no_evil: .

But I can also add an `is_string()` check there and leave it as is.